indra/assemblers/pybel/assembler.py

```diff
@@ -447,8 +447,10 @@ def _get_agent_node_no_bcs(agent):
     for mut in agent.mutations:
         var = hgvs(mut.to_hgvs())
         variants.append(var)
-
-    if variants:
+    
+    if variants and not isinstance(node_data, CentralDogma):
```

Comment:
This is interesting - looks like a valid concern but at least in the case of some modifications, they could apply to other things too. What kind of examples did you run into where this happened?

Reply:
Good question: I only found instances of GO, CHEBI, and PUBCHEM agents that had phosphorylation or acetylation modifications.

Reply:
I didn't look into which GO term it was, though. That might have been interesting...